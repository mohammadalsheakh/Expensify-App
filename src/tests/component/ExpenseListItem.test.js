import React from "react";
import { shallow } from "enzyme";
import ExpenseListItem from "../../components/expenseListItem";
import myExpenses from "../fixture/myexpenses";

test('should test the expense item ', () => {
    const wrapper = shallow(< ExpenseListItem {...myExpenses[0]} />)
    expect(wrapper).toMatchSnapshot()
})
