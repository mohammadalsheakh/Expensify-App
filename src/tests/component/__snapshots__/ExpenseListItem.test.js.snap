// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test the expense item  1`] = `
<div>
  <Link
    to="edit/1"
  >
    <h3
      style={
        Object {
          "color": "red",
        }
      }
    >
      Gum
    </h3>
  </Link>
  <p>
    195
     - 
    0
  </p>
</div>
`;
